lib/xctest.js

```diff
@@ -0,0 +1,241 @@
+import { startHouseArrestService, startTestmanagerdService, startInstallationProxyService, startInstrumentService } from './services';
+import { INSTRUMENT_CHANNEL } from './instrument';
+import { TESTMANAGERD_CHANNEL } from './testmanagerd';
+import { DTXMessageAuxBuffer } from './instrument/headers';
+import { NSUUID, XCTestConfiguration } from './instrument/transformer/nskeyed';
+import { uuidV4 } from '@appium/support/build/lib/util';
+import { getOSVersion } from './utilities';
+import log from './logger';
+import B from 'bluebird';
+
+const { DAEMON_CONNECTION_INTERFACE } = TESTMANAGERD_CHANNEL;
+const XCTEST_CONFIGURATION_EXTENSION = '.xctestconfiguration';
+const XCODE_VERSION = 29;
```

Comment:
Sorry I might miss this Xcode version's meaning in past review comments. Is this a dummy Xcode version, or has some specific meaning? A bride comment will help future maintenance